controller/array_action/array_mediator_svc.py

```diff
@@ -237,6 +237,10 @@ def get_volume(self, volume_name, pool_id=None):
     def get_volume_name(self, volume_id):
         return self._get_vol_by_wwn(volume_id)
 
+    def get_volume_by_id(self, volume_id):
+        cli_volume = self._get_cli_volume(volume_id)
```

Comment:
I think we might be missing `_get_vol_by_wwn` decorator in svc mediator as well

Reply:
maybe, related to this scope?

Reply:
if this doesn't work on svc, then yes

Reply:
why wouldn't it work?

Reply:
see other volume_id usages

Reply:
I guess we cant check that now..

Reply:
I changed it to a method that used in other places that passed tests